assets/src/stories-editor/blocks/amp-story-page-attachment/attachment-opener.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RichText } from '@wordpress/block-editor';
+import { ENTER, SPACE } from '@wordpress/keycodes';
+
+const AttachmentOpener = ( { setAttributes, toggleAttachment, openText } ) => {
+	return (
+		<div className="open-attachment-wrapper">
+			<span
+				role="button"
+				tabIndex="0"
+				onClick={ () => {
+					toggleAttachment( true );
+				} }
+				onKeyDown={ ( event ) => {
+					if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
```

Comment:
Nice detail, was very handy in my testing!

I wonder if there could be similar functionality when the attachment is already expanded. I kept pressing <kbd>ESC</kbd>, but all that happened was that the attachment block received focus:

![Screenshot 2019-09-02 at 12 07 01](https://user-images.githubusercontent.com/841956/64107346-97eb5f80-cd7a-11e9-8c85-f376b6fa206d.png)

Reply:
Hmm. We could try adding the `ESC` handling when Page Attachment is selected, however, this might create some confusion since if the block loses focus the Attachment will still stay open, and then `ESC` would not close the Attachment, the user might not notice that the block lost focus though. Probably catching any `ESC` event to close the attachment even if it's not selecting wouldn't be that great either.

Thoughts?

Reply:
Good points. Let's leave it as-is for now and see how it works during testing/beta.